packages/@o3r/rules-engine/src/engine/operator/operators/array-based.operators.ts

```diff
@@ -119,7 +119,11 @@ export const allLower: Operator<number[], number | string> = {
 export const allMatch: Operator<string[], string> = {
   name: 'allMatch',
   evaluator: (array, inputString) => {
-    const regExp = new RegExp(inputString);
+    let regExp = new RegExp(inputString);
+    if (inputString[0] === '/') {
```

Comment:
What if you want to match an url, wouldn't it detect it as a regex? Ex: /this/is/a/url.com

Would that be a valid scenario?

Reply:
No since slashes are special characters in a regex so it would need an escape character before
Ex: \/this\/is\/a\/url.com